text/0015-the-road-to-ember-2-0.md

```diff
@@ -1,10 +1,19 @@
 ---
-stage: accepted
+stage: discontinued # FIXME: Is this correct?
```

Comment:
We could also leave this as recommended. Not quite clear how we want to handle this sort of thing.